models/components/versions/parse.go

```diff
@@ -37,12 +40,16 @@ func ParseComponent(componentData []byte, fileName string) (base.Component, erro
 		c := new(v3.Component)
 		err = yaml.Unmarshal(componentData, c)
 		component = c
+	case ComponentV3_1_0.EQ(b.SchemaVersion):
```

Comment:
Need a test for this: https://codecov.io/gh/opencontrol/compliance-masonry/compare/d140f749617e3f5d8427f4df5e37ad32c57ee96d...2ef2863eee20aad0b7f0f7a2b83aff3990790eac?src=pr#6D6F64656C732F636F6D706F6E656E74732F76657273696F6E732F70617273652E676F-43